client/lib/generate-password/index.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+
```

Comment:
Nit: This empty line is unnecessary.

```suggestion
```

Reply:
fixed in https://github.com/Automattic/wp-calypso/pull/48762/commits/2c1fc92b1cbd60872d01e22380933f382b6d5e2d